### PR TITLE
feat(components): changed icons to represent delete and match mockup …

### PR DESCRIPTION
**Pull request by akonsowski**
… also icons are no longer clickable in listing management #620 
@wski 
@SOAP1914 
@rkenyon1969 
@clarkjefcoat

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -200,7 +200,7 @@ button.btn.image-remove {
 
     > .icon-trash {
 
-        @extend .icon-trash-grayDark;
+        @extend .icon-trash-12-blueDarker;
```
Comment (wski) [inline app/styles/main.scss]: this shouldn't be done, this is affecting global styles.

**Comment by wski** [comment]
:+1:

**Comment by SOAP1914** [comment]
👍  looks good to me

**Comment by rkenyon1969** [comment]
:+1:

**Comment by rkenyon1969** [comment]
sorry, wrong PR.

**Pull request merged**